### ISSUE TITLE
Can't work when upgrade  Deno to 1.13.0

### ISSUE THREAD

**Issue by qiuzhanghua** [eveningkid/denodb#289]
fille index.ts as:
```ts
import { Database, SQLite3Connector } from 'https://deno.land/x/denodb/mod.ts';

const connector = new SQLite3Connector({
  filepath: './database.sqlite',
});

const db = new Database(connector);

console.log(db);
```
with
```bash
deno cache index.ts
```

output:

```raw
Check file:///Users/q/deno/first/index.ts
error: TS2416 [ERROR]: Property 'name' in type 'PartialReadError' is not assignable to the same property in base type 'UnexpectedEof'.
  Type 'string' is not assignable to type '"UnexpectedEof"'.
  name = "PartialReadError";
  ~~~~
    at https://deno.land/std@0.74.0/io/bufio.ts:27:3

TS2416 [ERROR]: Property 'name' in type 'PartialReadError' is not assignable to the same property in base type 'UnexpectedEof'.
  Type 'string' is not assignable to type '"UnexpectedEof"'.
  name = "PartialReadError";
  ~~~~
    at https://deno.land/std@0.83.0/io/bufio.ts:27:3

Found 2 errors.
```

**Comment by jackfiszr** [comment]
This is a problem with:

mysql@v2.8.0 (it uses std@0.74.0)

mongo@v0.22.0 (it uses std@0.83.0)

**Comment by qiuzhanghua** [comment]
Can I exclude mysql and/or mango driver as I am using sqlite3 only?

**Comment by stillalivx** [comment]
You can use my branch as a temporary solution. Here it is: https://github.com/stillalivx/denodb
```typescript`
import { ... } from "https://raw.githubusercontent.com/stillalivx/denodb/master/mod.ts"
``

**Comment by qiuzhanghua** [comment]
Great!

**Comment by jackfiszr** [comment]
I think this should be open until the fix is merged and released.

**Comment by stillalivx** [comment]
@jackfiszr, you're right!
@qiuzhanghua could you reopen the issue, please? 😃

**Comment by chovy** [comment]
same issue for me

**Comment by chovy** [comment]
it is supposedly fixed in latest version of mysql

**Comment by eveningkid** [comment]
Hey,

Would switching `mysql` to 2.10.0 (latest) solve the issue?

Let me know

**Comment by jackfiszr** [comment]
@eveningkid Yes, mysql 2.10.0 uses std@0.104.0 which is OK with deno 1.13.0
Also, the errors do not accur when using deno 1.13.1

**Comment by eveningkid** [comment]
Thanks for the heads up @jackfiszr!

So, two things:
- upgrading to `mysql@2.10.0` worked fine
- upgrading to `mongo@0.24.0` raised the `missing --unstable flag` error for every single user (since they're using unstable APIs in 0.24.0). This means that you'd need this flag, even if you aren't using the mongo driver.  This has been an issue in the past and denodb will only rely on stable APIs for now

The patch will be available in `denodb@1.0.39`. Remember to call `--reload` in your command after upgrading:

```
deno run --reload script.ts
```

Thanks again to everyone who helped resolving this issue.

Have a great day :)

**Comment by ZeroThe2nd** [comment]
While this issue is now fixed for MySQL, I'm still running into it when using MongoDB on Deno v1.13.0. Is progress for this tracked somewhere?
